packages/twenty-server/src/engine/core-modules/billing/billing.resolver.ts

```diff
@@ -158,4 +174,42 @@ export class BillingResolver {
 
     return plans.map(formatBillingDatabaseProductToGraphqlDTO);
   }
+
+  private async validateCanCheckoutSessionPermissionOrThrow({
+    workspaceId,
+    userWorkspaceId,
+  }: {
+    workspaceId: string;
+    userWorkspaceId: string;
+  }) {
+    const isPermissionsEnabled = await this.featureFlagService.isFeatureEnabled(
+      FeatureFlagKey.IsPermissionsEnabled,
+      workspaceId,
+    );
+
+    if (isPermissionsEnabled) {
```

Comment:
prefer early return style

if (!isPermissionEnabled) {
   return true
}

if (...)

this avoids nesting ifs